src/sensor/ping360.cpp

```diff
@@ -73,6 +73,20 @@ Ping360::Ping360()
 
 void Ping360::startPreConfigurationProcess()
 {
+    // Stop all configuration/message timers if link is not writable
+    if(!checkForWitableLinkAndDisableConfiguration()) {
```

Comment:
```suggestion
    if(!checkForNonWritableLinkAndDisableConfiguration()) {
```

Reply:
Done.

Reply:
Ok, this doesn't look right, now. Did you change other stuff too?
Now the name does not agree with the returns! Sorry if this was my fault.

Reply:
I Changed the name again, for isWritable. This make it explicit what is happening.
Mostly of the cases we check for nonWritable, but the negative logic returned from a function makes it harder to understand since it's more implicit what is happening.